src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int a = 0;a < 3;a++) {
```

Comment:
```suggestion
        for (int a = 0; a  <  3; a++) {
```
Be attentive to spaces 
[All magic numbers in your code should be constants.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#all-magic-numbers-in-your-code-should-be-constants)